e2e-polybft/property_test.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+	"pgregory.net/rapid"
+)
+
+// property based tests enabled
+const envPropertyBaseTestEnabled = "E2E_PROPERTY_TESTS"
+
+func TestProperty_DifferentVotingPower(t *testing.T) {
+	if strings.ToLower(os.Getenv(envPropertyBaseTestEnabled)) != "true" {
```

Comment:
Should we export the function `isTrueEnv` from `test-cluster` and use that one instead?